src/lib/services/transfer/index.ts

```diff
@@ -19,3 +19,6 @@ export * as fileOperations from './service/file-operations.js';
 export * as taskSubmission from './service/task-submission.js';
 export * as endpoint from './service/endpoint.js';
 export * as task from './service/task.js';
+export * as access from './service/access.js';
+
+export * as endpointManager from './service/endpoint-manager';
```

Comment:
```suggestion
export * as endpointManager from './service/endpoint-manager.js';
```

Reply:
I get a lot of the following when I add the extension:
![image](https://github.com/globus/globus-sdk-javascript/assets/2126689/17ae480b-4047-4113-a103-77e55f0df47e)

Reply:
Ah, you need to include the full relative path: `export * as endpointManager from './service/endpoint-manager/index.js';` (required by ESM).

Reply:
:facepalm: